applications/sam/sam_do.py

```diff
@@ -583,6 +583,10 @@ def run(self, parser):
                 self.logger.error('Tuttle graph computation has failed.')
```

Comment:
Indeed. Currently the continueOnError option is only used by the host, but it could also be used by the command line. See commit 72cd31f